services/github/github-issues.service.js

```diff
@@ -25,81 +25,194 @@ module.exports = class GithubIssues extends LegacyService {
   static get route() {
     return {
       base: 'github',
+      pattern:
+        ':which(issues|issues-closed|issues-pr|issues-pr-closed)-:raw(raw)?/:user/:repo/:label?',
     }
   }
 
   static get examples() {
     return [
       {
         title: 'GitHub issues',
-        previewUrl: 'issues/badges/shields',
-        keywords: ['GitHub', 'issue'],
+        pattern: 'issues/:user/:repo',
+        namedParams: {
+          user: 'badges',
+          repo: 'shields',
+        },
+        staticPreview: {
+          label: 'issues',
+          message: '167 open',
+          color: 'yellow',
+        },
+        keywords: ['issue'],
         documentation,
       },
       {
         title: 'GitHub issues',
-        previewUrl: 'issues-raw/badges/shields',
-        keywords: ['GitHub', 'issue'],
+        pattern: 'issues-raw/:user/:repo',
+        namedParams: {
+          user: 'badges',
+          repo: 'shields',
+        },
+        staticPreview: {
+          label: 'open issues',
+          message: '167',
+          color: 'yellow',
+        },
+        keywords: ['issue'],
         documentation,
       },
       {
-        title: 'GitHub pull requests',
-        previewUrl: 'issues-pr/cdnjs/cdnjs',
-        keywords: ['GitHub', 'pullrequest', 'pr'],
+        title: 'GitHub issues by-label',
+        pattern: 'issues-raw/:user/:repo/:label',
```

Comment:
should this begin with `issues/:user/...`? if not what's the difference between this example and the next one aside from the value of the label being used?